lib/membrane/rtp/session_bin.ex

```diff
@@ -219,10 +230,10 @@ defmodule Membrane.RTP.SessionBin do
         spec: [rtp_extension_options_t()],
         default: [],
         description: """
-        List of RTP extension options. Currently `:vad` and `:twcc` are supported.
+        List of RTP extension options. Currently `:vad` and `:twcc` (receiver) are supported.
```

Comment:
This description is not enough to inform that only `{:twcc, %TWCCReceiver{}}` is allowed, IMO it should be more specific (or we can get rid of that key end enumerate just the modules :wink: )